src/javascripts/_helpers.js

```diff
@@ -0,0 +1,55 @@
+var UK_Parliament = (function(){
```

Comment:
Redeclaring this variable in the other files means that you're overwriting the previous module?